wenet/transformer/encoder.py

```diff
@@ -250,6 +250,58 @@ def forward_chunk(
         return (xs[:, cache_size:, :], r_subsampling_cache,
                 r_elayers_output_cache, r_conformer_cnn_cache)
 
+    def forward_chunk_v2(
+        self,
+        xs: torch.Tensor,
+        infer_control: torch.Tensor,
+        cnn_cache: torch.Tensor,
+        key_cache: torch.Tensor,
```

Comment:
seems key_cache and value cache shares the same operation and interface, could they be combined?